index.html

```diff
@@ -262,7 +262,30 @@ <h3 id="unacceptablebehavior">
             <a>Harassing</a> photography or recording, including logging online
             activity for <a>harassment</a> purposes.
           </li>
-          <li>Sustained disruption of discussion.
+          <li>Sustained disruption of discussion. This may include, but
+          is not lmited to, various common methods of engaging in bad
```

Comment:
s/lmited/limited

Reply:
Thank you so much @hober. This is extremely helpful!

Reply:
Fixed in 5ce7e84.